R/derive_var_basetype.R

```diff
@@ -7,7 +7,8 @@
 #'
 #'   The columns specified in the expressions inside `basetypes` are required.
 #'
-#' @param basetypes A *named* list of expressions created using `exprs()`
+#' @param basetypes A *named* list of expressions created using
+#' [`exprs()`](https://rlang.r-lib.org/reference/defusing-advanced.html)
 #'
```

Comment:
Can we just cheat and say use the function `exprs` and link out to documentation.  

I'm also confused.  Is it executing that as like an example??

Reply:
happy to do that and take out any link - will update. seems easiest fix as anyone could go and google it